style.css

```diff
@@ -1,3 +1,126 @@
-#root {
-  color: red;
+/* ------- Mobile View --------- */
+
+body {
+  display: flex;
+  flex-direction: column;
+ 
+}
+
+header {
+  text-align: center;
+  background: conic-gradient(at 98% 10%, red 20%, yellow 60%, #1e90ff 100%);
+}
+#h1 {
+  font-size: xx-large;
+}
+
+#search_section {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  font-size: larger;
+}
+
+#number_of_episodes,
+h1 {
+  font-style: italic;
+  font-size: larger;
+  font-weight: bolder;
+  background-image: linear-gradient(to top, #0fd850 0%, #f9f047 100%);
+  border-radius: 25%;
+}
+
+h2 {
+  font-size: 1.5vw;
+  text-align: center;
+}
+main {
+  display: flex;
+  flex-direction: column;
+  border: solid;
+  align-items: center;
+  background: linear-gradient(
+      217deg,
+      rgba(255, 0, 0, 0.8),
+      rgba(255, 0, 0, 0) 0.71%
+    ),
+    linear-gradient(127deg, rgba(0, 255, 0, 0.8), rgba(0, 255, 0, 0) 70.71%),
+    linear-gradient(336deg, rgba(0, 0, 255, 0.8), rgba(0, 0, 255, 0) 120.71%);
+}
+
+.article_class {
+  width: 25vw;
+  height: 85%;
+  padding: 20px;
+  margin: 6px;
+  display: flex;
+  flex-direction: column;
+  border-radius: 5px;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+  background: conic-gradient(at 0% 15%, red 10%, yellow 30%, #1e90ff 50%);
+  transition: transform 0.5s;
+}
+
+.p_class {
+  grid-area: p;
+  font-size: 1.2vw;
+  margin-left: 5px;
+  width: 100%;
+}
+.article_class:hover {
+  /*transform: scale(2); IE 9*/
+  transform: scale(1.2); /* Safari 3-8 */
+}
+
+#rights,
+a {
+  color: white;
+  font-size: x-large;
+  text-align: center;
+}
+footer {
+  width: 100%;
+  height: 80px;
+  background-color: rgb(56, 47, 79);
+}
+
+/* Medium devices eg: tablets */
+@media only screen and (min-width: 426px) and (max-width: 700px) {
```

Comment:
Responsive design! Mobile first too! This is brillant; even the solution doesn't have it. Glad you haven't forgottonn your CSS!